### PR TITLE
fix: mailer integration test & wallet binding

### PR DESCRIPTION
**Pull request by rubenkristian**
### JIRA LINK
- https://blocksphere2020.atlassian.net/browse/DBIO-2508

### Changelog
- fix integration test mailer with remove './' from template
- fix wallet binding
- remove unused variable
- change type
- update package
- update unit test
- update setting integration test

### CODE REVIEW

**Comment by rubenkristian** [comment]
image

File: .github/workflows/tester.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - name: Install Dependencies
-        run: npm install
+        run: npm install --force
```
Comment (agustinustheo) [inline .github/workflows/tester.yml:29]: Why use `--force`?

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: .github/workflows/tester.yml (hunk 1 above)
Comment (rubenkristian) [reply .github/workflows/tester.yml:29]: Install Dependencies get error on Github Action.

but when i tried it locally i don't get any error, i used same version nodejs.

Comment (rubenkristian) [reply .github/workflows/tester.yml:29]: i add --force after get solution from a website

**Comment by rubenkristian** [comment]
image

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**